### PR TITLE
Update telemetryGenerator version & add more error log

### PR DESCRIPTION
**Pull request by willyyhuang**
## Types of changes

- [❌] Bug fix (non-breaking change which fixes an issue)
- [✅  New feature (non-breaking change which adds functionality)

## Description
- Updated telemetry generator version
- Added more error log into createUpload metric 

## Checklist
- [✅] My code follows the code style of this project
- [❌] I have added tests to cover my changes
- [❌] A short description of the change has been added to the **[CHANGELOG](https://github.com/aws/aws-toolkit-jetbrains/blob/master/CONTRIBUTING.md#contributing-via-pull-requests)** if the change is customer-facing in the IDE.
- [✅] I have added metrics for my changes (if required)
 
## License
I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by rli** [comment]
`[x]` is sufficient for a check box in the PR description if you get tired of looking up the emojis

**Comment by willyyhuang** [comment]
> `[x]` is sufficient for a check box in the PR description if you get tired of looking up the emojis

lol i don't really mind adding these emojis, control + command + space opens up an emoji menu

File: plugins/toolkit/jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/actions/GoToHandlerAction.kt [hunk 1]
```diff
@@ -30,9 +30,9 @@ class GoToHandlerAction : ExplorerNodeAction<LambdaFunctionNode>(ActionsBundle.a
         val handlers = getHandler(selected.first())
         if (handlers != null) {
             OpenSourceUtil.navigate(true, *handlers)
-            LambdaTelemetry.goToHandler(e.project, true)
+            LambdaTelemetry.goToHandler(project = e.project, passive = true)
```
Comment (willyyhuang) [inline plugins/toolkit/jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/actions/GoToHandlerAction.kt:33]: not exactly sure which metadata this boolean is supposed to be mapped to...

**Review by chengoramazon — APPROVED** [review]

File: plugins/toolkit/jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/actions/GoToHandlerAction.kt (hunk 1 above)
Comment (rli) [inline plugins/toolkit/jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/actions/GoToHandlerAction.kt:33]:
```suggestion
            LambdaTelemetry.goToHandler(project = e.project, success = true)
```

File: plugins/toolkit/jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/actions/GoToHandlerAction.kt [hunk 2]
```diff
@@ -30,9 +30,9 @@ class GoToHandlerAction : ExplorerNodeAction<LambdaFunctionNode>(ActionsBundle.a
...
         } else {
-            LambdaTelemetry.goToHandler(e.project, false)
+            LambdaTelemetry.goToHandler(project = e.project, passive = false)
```
Comment (rli) [inline plugins/toolkit/jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/actions/GoToHandlerAction.kt:35]:
```suggestion
            LambdaTelemetry.goToHandler(project = e.project, success = false)
```

**Review by rli — APPROVED** [review]

**Pull request merged**